app/views/RoomActionsView/index.js

```diff
@@ -770,7 +745,7 @@ class RoomActionsView extends React.Component {
 					<List.Item
 						title='Leave'
 						onPress={() => this.onPressTouchable({
-							event: room.teamMain ? this.leaveTeam : this.leaveChannel
+							event: room.teamMain ? this.leaveTeamRoom : this.leaveChannel
```

Comment:
`this.leaveTeamRoom` doesn't exist

Reply:
Yes, my fault when I fixed the last conflict. Now it's fixed!